llm-complete-guide/README.md

```diff
@@ -0,0 +1,144 @@
+# 🦜 Production-ready RAG pipelines for chat applications
+
+This project showcases how you can work up from a simple RAG pipeline to a more complex setup that
+involves finetuning embeddings, reranking retrieved documents, and even finetuning the
+LLM itself. We'll do this all for a use case relevant to ZenML: a question
+answering system that can provide answers to common questions about ZenML. This
+will help you understand how to apply the concepts covered in this guide to your
+own projects.
+
+![](.assets/rag-pipeline-zenml-cloud.png)
+
+Contained within this project is all the code needed to run the full pipelines.
+You can follow along [in our guide](https://docs.zenml.io/user-guide/llmops-guide/) to understand the decisions and tradeoffs
+behind the pipeline and step code contained here. You'll build a solid understanding of how to leverage
+LLMs in your MLOps workflows using ZenML, enabling you to build powerful,
+scalable, and maintainable LLM-powered applications.
+
+This project contains all the pipeline and step code necessary to follow along
+with the guide. You'll need a PostgreSQL database to store the embeddings; full
```

Comment:
is it possible to use postgressql directly as a docker image or something or is supabase needed? Because the variables exported are not postgres variables but rather supabase ones

Reply:
Yeah I can clear that up. Actually it will work with any Postgres database, so I should just rename the variables needed.